packages/elements/src/components/file-picker/file-picker.component.ts

```diff
@@ -0,0 +1,279 @@
+    this.fileIndex = 0;
+  }
+
+  connectedCallback(): void {
+    super.connectedCallback();
+
+    this.#prepareMap();
+    this.#filter('');
+
+    window.addEventListener('keydown', (e) => this.#handleKeyDown(e), { signal: this.#abortController.signal });
+  }
+
+  disconnectedCallback(): void {
+    super.disconnectedCallback();
+
+    this.#abortController.abort();
+  }
+
+  updated(changedProperties: Map<string | number | symbol, unknown>): void {
+    super.updated(changedProperties);
+
+    if (changedProperties.has('rootModel')) {
+      this.#prepareMap();
+    }
+  }
+
+  open() {
+    this.openPicker = true;
+
+    void this.updateComplete.then(() => this.#focusInput());
+  }
+
+  render() {
+    if (!this.openPicker) {
+      return nothing;
+    }
+
+    return html`
+      <div
+        id="backdrop"
+        @click="${() => this.#closePicker()}"
+        class="fixed left-0 top-0 z-50 flex h-full w-full justify-center bg-black/50 backdrop-blur-lg"
+      >
+        <div
+          @click="${(e: MouseEvent) => e.stopPropagation()}"
+          role="dialog"
+          id="picker"
+          class="m-4 flex h-fit max-h-[500px] w-full max-w-[40rem] flex-col rounded-lg bg-gray-200/60 p-4 backdrop-blur-lg md:w-1/2"
+        >
+          <div class="mb-2 flex items-center rounded bg-gray-200/60 p-2 backdrop-blur-lg">
+            <div class="mx-2 flex items-center text-gray-800">${searchIcon}</div>
+            <label for="file-picker-input" class="sr-only">Search for a file</label>
+            <input
+              id="file-picker-input"
+              @keyup="${(e: KeyboardEvent) => this.#handleSearch(e)}"
+              type="text"
+              style="box-shadow: none"
+              class="mr-2 w-full rounded border-0 border-transparent bg-transparent text-gray-800 focus:shadow-none"
+              placeholder="Search for a file"
+            />
+          </div>
+          <div tabindex="-1" class="overflow-auto"> ${this.#renderFoundFiles()} </div>
+        </div>
+      </div>
+    `;
+  }
+
+  #renderFoundFiles() {
+    return html`
+      <ul id="files" class="flex flex-col">
+        ${renderIf(this.filteredFiles.length === 0, () => html`<li class="text-gray-800">No files found</li>`)}
+        ${repeat(
+          this.filteredFiles,
+          (item) => item.name,
+          ({ name, file }, index) => {
+            return html` <li>
+              <a
+                ?data-active="${index === this.fileIndex}"
+                tabindex="${index === this.fileIndex ? 0 : -1}"
+                @focusout="${() => this.#handleFocus()}"
+                @click="${() => this.#closePicker()}"
+                class="${classMap({
+                  'border-primary-500': index === this.fileIndex,
+                })} my-1 flex rounded border-2 border-white bg-white p-1 px-2 text-gray-800 outline-none focus-visible:border-primary-200"
+                href="${toAbsoluteUrl(this.#getFragment(file), file.name)}"
```

Comment:
```suggestion
                href="${toAbsoluteUrl(this.#getFragment(file), name)}"
```

If I press Enter it goes to the correct file, but clicking on the link doesn't